searchbar.js

```diff
@@ -4,45 +4,66 @@ var lessonOneContent = lessonOne.textContent.toLowerCase();
 var lessonOneArr= lessonOneContent.split(/\W/g)
 var lessonTwo = document.getElementById("lesson-two");
 var lessonTwoContent = lessonTwo.textContent.toLowerCase();
-var lessonTwoArr= lessonTwoContent.split(/\W/g)
+var lessonTwoArr = lessonTwoContent.split(/\W/g)
+var noMatchesBanner = document.getElementById("noMatchesBanner")
 //use split on the string, instead of checking is equal, check if text input is within the sting (sub-sting) 
 
 function searchFunction(e) {
     var text = e.target.value.toLowerCase();
     let isLessonOneAMatch = false;
     let isLessonTwoAMatch = false;
     let userInput = text.split(/\W/g);
-    console.log(userInput)
-    console.log(lessonOneArr)
-    console.log(lessonTwoArr)
+    // console.log(userInput)
```

Comment:
Try not to leave commented code in commits! My strong preference is just never to comment out code, even if you think you might want it back later.

I think it's more valuable in building muscle memory with your language to retype a line if you think you need it back, plus it makes you think about what code you _really_ want, instead of just falling back to whatever you had before. 

If you _really_ want to save some code to put back in later, cut/paste it to an empty file (people sometimes call this a scratch file) instead of commenting.